docs/pages/reference/cli.mdx

```diff
@@ -1,53 +1,68 @@
 ---
 title: Teleport CLI Reference
 description: Detailed guide and reference documentation for Teleport's command line interface (CLI) tools.
-h1: Command Line (CLI) Reference
 ---
 
-Teleport is made up of three CLI tools.
+Teleport is made up of four CLI tools.
 
-- [teleport](#teleport): The daemon that runs the Auth Service, Proxy Service, and other Teleport services.
-- [tsh](#tsh): A tool that lets end users interact with Teleport.
-- [tctl](#tctl): An administrative tool that can configure the Teleport Auth Service.
+- [teleport](#teleport): A CLI tool that supports the Teleport access plane by starting and configuring various Teleport services.
```

Comment:
I don't think we need to repeat "CLI tool" in each list item, since we already say that each list item describes a CLI tool.